ignite/services/network/peer.go

```diff
@@ -28,3 +39,63 @@ func ParsePeerAddress(addr string) (launchtypes.Peer, error) {
 	}
 	return launchtypes.NewPeerConn(sp[0], sp[1]), nil
 }
+
+const TunnelRerunDelay = 5 * time.Second
+
+// StartProxyForTunneledPeers starts an HTTP proxy server and HTTP proxy clients
+// for each node that needs HTTP tunneling.
+// HTTP tunneling is activated **ONLY** if your app's `$APP_HOME/config` dir has an `spn.yml` file
+// and only if this file has `tunneled_peers` field inside with a list of tunneled peers/nodes.
+//
+// If you're using SPN as coordinator and do not want to allow HTTP tunneling feature at all,
+// you can prevent `spn.yml` file to being generated by not approving validator requests
+// that has HTTP tunneling enabled instead of plain TCP connections.
+func StartProxyForTunneledPeers(ctx context.Context, clientCtx client.Context, serverCtx *server.Context) {
```

Comment:
I wasn't sure where to put this function (intially it was in `pkg/cosmoscmd`), because it depends on `spn` I put it there but maybe it's not the best location.

Reply:
I think this location makes sense!